config/conversations/questions/whats_for_lunch.rb

```diff
@@ -0,0 +1,99 @@
+require "net/https"
```

Comment:
You require `"net/https"`, but I don't think you use it — I think you just use HTTP, no "s"